modules/oe_webtools_cookie_consent/oe_webtools_cookie_consent.module

```diff
@@ -29,7 +30,21 @@ function oe_webtools_cookie_consent_page_attachments(array &$attachments) {
   $cache = $event_cache->merge($existing_cache);
   $cache->applyTo($attachments);
   if ($event->isBannerPopup()) {
-    $attachments['#attached']['library'][] = 'oe_webtools_cookie_consent/oe_webtools_cookie_consent.cck';
```

Comment:
If we no longer use the custom library we should remove it.

Reply:
Thanks. Fixed.

Reply:
So change of plans (sorry @sergepavle :cry: ), lets make this backwards compatible by flagging the library as deprecated. We can remove it in the next major release.